journal/method_body.hpp

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include "journal/method.hpp"
+#include "journal/recorder.hpp"
```

Comment:
Lint says: `Found C++ system header after other header. Should be: method_body.h, c system, c++ system, other.`

This include should go after the include of glog/logging.h at line 8.